mta/tests/test_global_rules.py

```diff
@@ -111,42 +113,47 @@ def test_analysis_global_custom_rule(application, add_global_custom_rule, create
     view.application_table.application_details("acmeair-webapp-1.0-SNAPSHOT.war")
     view.tabs.issues.click()
     view = analysis_results.create_view(Issues)
-    # TODO(ghubale): Updated test case with reading Migration potential table
+    # TODO(ghubale): Update test case with reading Migration potential table
     assert view.wait_displayed
 
 
 def test_invalid_rule_file_type(application, request):
-    """ Test to upload global_1 custom rules file
+    """ Test to upload global custom rules file
 
     Polarion:
         assignee: ghubale
         initialEstimate: 1/12h
         caseimportance: medium
         testSteps:
-            1. Upload invalid global_1 custom rule file
+            1. Upload invalid global custom rule file
             2. Check number of rules in it
         expectedResults:
-            1. Invalid global_1 custom rule file should have 0 rules
+            1. Invalid global custom rule file should have 0 rules
     """
-    file_name = "customWebLogic.windup.label.xml"
-    rules_configurations = CustomRulesConfiguration(application, file_name)
-    rules_configurations.upload_custom_rule_file()
-    view = rules_configurations.create_view(CustomRulesView)
-    view.table.wait_displayed("20s")
+    file_names = ["customWebLogic.windup.label.xml", "empty_rule_file.xml"]
 
     @request.addfinalizer
     def _finalize():
-        view.search.fill("")
-        rules_configurations.delete_custom_rule()
+        for file in file_names:
+            rules = CustomRulesConfiguration(application, file)
+            rules.delete_custom_rule()
 
-    all_rules = view.table.read()
-    for rule in all_rules:
-        if rule["Short path"] == file_name:
-            assert int(rule["Number of rules"]) == 0
+    for file_name in file_names:
+        rules_configurations = CustomRulesConfiguration(application, file_name)
+        rules_configurations.upload_custom_rule_file()
+        view = rules_configurations.create_view(CustomRulesView)
+        view.table.wait_displayed("20s")
+
+        all_rules = view.table.read()
+        for rule in all_rules:
+            if rule["Short path"] == file_name:
+                assert int(rule["Number of rules"]) == 0
+    else:
```

Comment:
what is this for ?
assert False ?

Reply:
Case:
`all_rules` list empty. In this case, for loop fails and test has to FAIL.